roles/ceph-osd/tasks/check_mandatory_vars.yml

```diff
@@ -66,6 +73,18 @@
     - raw_journal_devices|length == 0
       or devices|length == 0
 
+- name: make sure the raw_journal_devices variable is a list
+  fail:
+    msg: "raw_journal_devices: must be a list, not a string, i.e. [ \"/dev/sda\" ]"
+  when:
+    - osd_group_name is defined
+    - osd_group_name in group_names
+    - not containerized_deployment
+    - raw_multi_journal
```

Comment:
just to make sure the `or` doesn't get messed up, let's move `raw_journal_devices is string` after `raw_multi_journal`.